controller/lib/core/actuators.h

```diff
@@ -21,15 +21,17 @@ limitations under the License.
 
 #include "blower_fsm.h"
 #include "hal.h"
+#include <optional>
 
 struct ActuatorsState {
-  float fan_setpoint_cm_h2o = 0.0;
-  ValveState expire_valve_state = ValveState::CLOSED;
-  float fan_power = 0.0;
-  float fan_valve = 0.0f;
+  float setpoint_cm_h2o{0.0f};
+  float fio2_valve{0.0f};
```

Comment:
Could you add some comments explaining the ranges and direction of these values (e.g. does 0.0 of fio2_valve mean it's open or closed?), and what does a nullopt mean in the optional ones?